django/core/handlers/base.py

```diff
@@ -82,6 +144,28 @@ def get_response(self, request):
             )
         return response
 
+    async def get_response_async(self, request):
```

Comment:
Might this be more at home in ASGIHandler?

Reply:
I didn't want to put it there in case someone wanted to implement an async-capable handler that was not based on ASGI. It's the same reasoning `get_response` is here, not in WSGIHandler.